packages/framework/presence/package.json

```diff
@@ -79,12 +79,14 @@
 		"format:prettier": "prettier --write . --cache --ignore-path ../../../.prettierignore",
 		"lint": "fluid-build . --task lint",
 		"lint:fix": "fluid-build . --task eslint:fix --task format",
-		"test": "npm run test:mocha",
-		"test:coverage": "c8 npm test",
+		"test": "npm run test:mocha && npm run test:vitest",
+		"test:coverage": "c8 npm run test:mocha && npm run test:vitest:coverage",
 		"test:mocha": "npm run test:mocha:esm && npm run test:mocha:cjs",
 		"test:mocha:cjs": "mocha --recursive \"dist/test/**/*.spec.*js\" --exit",
 		"test:mocha:esm": "mocha --recursive \"lib/test/**/*.spec.*js\" --exit",
 		"test:mocha:verbose": "cross-env FLUID_TEST_VERBOSE=1 npm run test:mocha",
+		"test:vitest": "vitest run",
+		"test:vitest:coverage": "npm run test:vitest -- --coverage",
```

Comment:
What does this actually test against? Is it doing ESM? How do I know?

Reply:
It's the "include" setting in the vitest.config.ts file. Right now it's running against the src but we can switch to lib either in config or with a CLI flag. Do you want lib all the time or lib in CI, src locally?

Reply:
Well, I would say lib almost all of the time. It can run against src as a special local dev case.

Reply:
It now runs against both src and lib, and outputs the snapshots for both versions of the test in the src/test/__snapshots__ folder. The tests run fast enough that running both isn't a big deal at the moment. I think we can further refine the config, but that can be done separately.